### PR TITLE
chore: refactor nodejs-repo-tools and stubbing

### PR DESCRIPTION
**Pull request by sofisl**

### CODE REVIEW

File: functions/helloworld/test/sample.integration.storage.test.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ describe('functions_helloworld_storage integration test', () => {
 
   it('helloGCS: should print metadata updated message', async () => {
     const filename = uuid.v4(); // Use a unique filename to avoid conflicts
-    const PORT = 9001; // Each running framework instance needs a unique port
+    const PORT = 8999; // Each running framework instance needs a unique port
```
Comment (ace-n) [inline functions/helloworld/test/sample.integration.storage.test.js]: What's the reason for this change?

Comment (sofisl) [reply functions/helloworld/test/sample.integration.storage.test.js]: Ah, I was getting an error saying that port was already in use when I ran the tests.

**Comment by ace-n** [comment]
LGTM for `functions` - thanks for clarifying @sofisl! 🙂
